docs/reference/overview/powerpoint-add-ins-reference-overview.md

```diff
@@ -0,0 +1,38 @@
+---
+title: JavaScript API for PowerPoint
+description: ''
+ms.date: 07/05/2019
+ms.prod: powerpoint
+localization_priority: Priority
+---
+
+# JavaScript API for PowerPoint
+
+A PowerPoint add-in interacts with objects in PowerPoint by using the JavaScript API for Office, which includes two JavaScript object models:
+
+* **PowerPoint JavaScript API**: Introduced with Office 2016, the [PowerPoint JavaScript API](/javascript/api/powerpoint) provides strongly-typed objects that you can use to access objects in PowerPoint. 
```

Comment:
There aren't actually any 2016 PPT APIs. There's a single createPresentation API that is a wrapper around Common API functionality (I believe).